Java/RollDice/README.md

```diff
@@ -0,0 +1,21 @@
+# RollDice  
+
+## Introduction
+```
+It is the program that will help you to Roll the Dice.
+```
+
+## Requirements
+```
+JDK
+JRE
```

Comment:
As the program runs in terminal so user can use any IDE of his choice to run this program. Only requirement should be Java IDE.